### ISSUE TITLE
small percent bad plots, Bladebit 3.1 --disk-128

### ISSUE THREAD

**Issue by jmhands** [Chia-Network/bladebit#393]
After creating 596 plots, `chia plots check -n 100` finds 2 plots that are bad.

```
2023-09-03T18:38:36.909  chia.plotting.check_plots        : INFO        Proofs 2 / 100, 0.02. Filepath: /dan/hddZR51G26E/plot-k32-c03-2023-09-01-22-40-7427409e2bc8a501cdf069679dbb3897d9bb9b88b33d5c40c13648e918fee097.plot
2023-09-03T20:18:54.832  chia.plotting.check_plots        : INFO        Proofs 21 / 100, 0.21. Filepath: /mnt/ssd/plot-k32-c03-2023-09-02-16-22-1929f0655c87a13b216a34172e0c565e49f6d732bbe1a66137c63b21e40ff781.plot
```

```
./bladebit_cuda check -n 1000 /mnt/badplots/plot-k32-c03-2023-09-01-22-40-7427409e2bc8a501cdf069679dbb3897d9bb9b88b33d5c40c13648e918fee097.plot
Checking 1000 random proofs with seed 0x7b3afd906424872eaabcbb70e24ebc3d91ebd79df2f4d508bea98e48a74434f2...
Plot compression level: 3
Using GPU for decompression.
 10%...
 20%...
 30%...
 40%...
 50%...
 60%...
 70%...
 80%...
 90%...
18 / 1000 (1.80%) valid proofs found.

./bladebit_cuda check -n 1000 /mnt/badplots/plot-k32-c03-2023-09-02-16-22-1929f0655c87a13b216a34172e0c565e49f6d732bbe1a66137c63b21e40ff781.plot
Checking 1000 random proofs with seed 0x4d3b507d67494d641e201cd5523fce89b7830c7aaa951469ae48cd4b6c9886e0...
Plot compression level: 3
Using GPU for decompression.
 10%...
 20%...
 30%...
 40%...
 50%...
 60%...
 70%...
 80%...
 90%...
189 / 1000 (18.90%) valid proofs found.
```

command ran was

`./bladebit_cuda -f -c --compress 3 -n 0 cudaplot --disk-128 -t1 /mnt/ssd/ /mnt/ssd`

plot ids are
```
7427409e2bc8a501cdf069679dbb3897d9bb9b88b33d5c40c13648e918fee097
1929f0655c87a13b216a34172e0c565e49f6d732bbe1a66137c63b21e40ff781
```

system configuration
SYSTEM_NAME="smci"
OS="Ubuntu 23.04 6.2.0-31-generic"
SYSVENDOR_MODEL="Supermicro Super Server"
CPU="1x AMD EPYC 7302 16-Core Processor (C:16|T:32)"
GPU="NVIDIA GeForce RTX 3060 Ti"
GPU_SKU="Ampere"
GPU_ARCH="900-1G142-2520-000"
GPU_DRIVER="530.30.02"
PCIE_LINK="PCIe 4x16"
DRAM=4x 16GB DDR4 ECC at 2133 MT/s, 36ASF2G72PZ-2G1B1
SSD=2x WUS4BA138DSP3X3 (WD SN840) in LVM RAID0, xfs

**Comment by jmhands** [comment]
[check.zip](https://github.com/Chia-Network/bladebit/files/12508124/check.zip)

**Comment by jmhands** [comment]
Total Proofs: 59,446
Total Challenges: 59,600
Proofs per Challenge: Approximately 0.9974

removing the 2 bad plots from the analysis
Mean Ratio: 1.0004
Standard Deviation: 0.098

![image](https://github.com/Chia-Network/bladebit/assets/69059890/fbc25b10-0c44-404a-9c76-a8927ca64d22)

**Comment by harold-b** [comment]
Fixed by #410
